edgekv/src/datastore.rs

```diff
@@ -574,7 +572,7 @@ impl DataStore {
 impl Drop for DataStore {
     fn drop(&mut self) {
         // TODO - TE-721: handle this error
-        self.sync_all(false);
+        let _ = self.sync_all(false);
```

Comment:
@binier `let _ =` will silence the warning but silently discard the error, something we should avoid

Reply:
There doesn't seem to be a way to obtain the current logger unless it is passed around and stored. What we have here is a common situation and it is kind of a PITA to solve because of that.

Reply:
Yes I agree. I just wanted to get rid of compilation warning.